Sources/KIF/Additions/UIAccessibilityElement-KIFAdditions.m

```diff
@@ -105,19 +115,24 @@ + (BOOL)accessibilityElement:(out UIAccessibilityElement **)foundElement view:(o
 }
 
 + (BOOL)accessibilityElement:(out UIAccessibilityElement *__autoreleasing *)foundElement view:(out UIView *__autoreleasing *)foundView withElementMatchingPredicate:(NSPredicate *)predicate fromRootView:(UIView *)fromView tappable:(BOOL)mustBeTappable error:(out NSError *__autoreleasing *)error
+{
+    return [self accessibilityElement: foundElement view:foundView withElementMatchingPredicate:predicate tappable:mustBeTappable error:error disableScroll:NO];
```

Comment:
Ok, I found the bug leading to the other set of tests that were failing in our repo, and it is caused by dropping this param here:

```suggestion
    return [self accessibilityElement:foundElement view:foundView withElementMatchingPredicate:predicate fromRootView:fromView tappable:mustBeTappable error:error disableScroll:NO];
```

Reply:
Sorry for that, it is just a mistake from my side. I've applied your suggestion! 🙏🏻